src/main/scala/li/cil/oc/common/tileentity/traits/RedstoneAware.scala

```diff
@@ -80,7 +80,7 @@ trait RedstoneAware extends RotationAware {
 
   def getOutput: Array[Int] = EnumFacing.values.map{ side: EnumFacing => _output(toLocal(side).ordinal) }
 
-  def getOutput(side: EnumFacing) = if (_output != null && _output.length <= toLocal(side).ordinal())
+  def getOutput(side: EnumFacing) = if (_output != null && _output.length >= toLocal(side).ordinal())
```

Comment:
Shouldn't this be `_output.length > toLocal(side).ordinal()` then?
Since `toLocal(side).ordinal()` is used as the index, so if it's `== _output.length` it'd be out of bounds?

Reply:
Yes, you're right.